lib/classes/PluginManager.js

```diff
@@ -346,9 +346,16 @@ class PluginManager {
     // Check if the command references an alias
     const aliasCommandTarget = this.getAliasCommandTarget(commandsArray);
     const commandOrAlias = aliasCommandTarget ? _.split(aliasCommandTarget, ':') : commandsArray;
+
     return _.reduce(commandOrAlias, (current, name, index) => {
-      if (name in current.commands &&
-         (allowEntryPoints || current.commands[name].type !== 'entrypoint')) {
+      const commandExists = name in current.commands;
+      const isNotContainer = commandExists && current.commands[name].type !== 'container';
+      const isNotEntrypoint = commandExists && current.commands[name].type !== 'entrypoint';
+      const remainingIterationsLeft = index < commandOrAlias.length - 1;
```

Comment:
Could you explain the need for `remainingIterationsLeft`?

Reply:
If the current command is a `container`, and there are remaining iterations, as in, more subcommands in `commandsArray` to process, then it continues looping.

For example, suppose mycommand below is a `container`, by checking there are "remainingIterationsLeft" I make sure the reducer checks for subcmd too.

`commandsArray: [ 'mycommand', 'subcmd' ]`

On the other hand, if you have:

`commandsArray: [ 'mycommand' ]`

There wouldn't be any iterations left, and also because is a `container` command is going to error with sls command not found.

Reply:
Ah! That makes sense. Thanks.